integration/plugin_manager/plugin_install_spec.rb

```diff
@@ -0,0 +1,43 @@
+# Encoding: utf-8
+require_relative "../spec_helper"
+require "fileutils"
+
+context "bin/plugin install" do
+  context "with a local gem" do
+    let(:gem_name) { "logstash-input-wmi" }
+    let(:local_gem) { gem_fetch(gem_name) }
+
+    it "install the gem succesfully" do
+      result = command("bin/plugin install --no-verify #{local_gem}")
+      expect(result.exit_status).to eq(0)
+      expect(result.stdout).to match(/Installing #{gem_name}/)
```

Comment:
no, no I even If my formal side would prefer it, I see the benefit of what you did here, no questions on it.

My point was on extending the match regular expression, now it looks for a sentence, but it does not care where is it printed, could be that ```Installing gem_name``` is not in the right place. Does it makes sense to check that too? what do you think?

Doing it would be as easy as making the expression a bit more complex, but not more.